example/src/App.tsx

```diff
@@ -37,6 +38,7 @@ const App: React.FC<{}> = () => {
           </PTZProvider>
         </Route>
         <Route path={`/tabs`} exact={true} component={TabsPage} />
+        <Route path={`/globalUI`} exact={true} component={GlobalUIComp} />
```

Comment:
We can do it as following:
`<Route path='/globalUI' component={GlobalUIComp} exact />`

No need to wrap string inside {}, we can ignore the true value for boolean props

Reply:
Fixed